store/migrations/migration1603803454/migrate.go

```diff
@@ -0,0 +1,51 @@
+package migration1603803454
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/external-initiator/store/migrations/migration0"
+	"github.com/smartcontractkit/external-initiator/store/migrations/migration1576509489"
+	"github.com/smartcontractkit/external-initiator/store/migrations/migration1576783801"
+	"github.com/smartcontractkit/external-initiator/store/migrations/migration1587897988"
+	"github.com/smartcontractkit/external-initiator/store/migrations/migration1592829052"
+)
+
+type EthQaeSubscription struct {
+	gorm.Model
+	SubscriptionId uint `gorm:"index"`
+	Address        string
+	ABI            string
+	ResponseKey    string
+	MethodName     string
+}
+
+type Subscription struct {
+	gorm.Model
+	ReferenceId       string `gorm:"unique;not null"`
+	Job               string
+	EndpointName      string
+	Ethereum          migration0.EthSubscription
+	Tezos             migration1576509489.TezosSubscription
+	Substrate         migration1576783801.SubstrateSubscription
+	Ontology          migration1587897988.OntSubscription
+	BinanceSmartChain migration1592829052.BinanceSmartChainSubscription
```

Comment:
I do not think this is used actually. The reasoning would be the same as the issue you mentioned in the comment below, that we separate initiators with different configurations. I do however think it would make sense to consolidate the initiators that have the same type of configurations.